### PR TITLE
Fix README

### LINKED ISSUE #2: RFC assignee and sub-team
These are referenced in the README without an explanation of how they come to be. We'll need to address that.

- Is the RFC assignee always going to be a maintainer?
- I take it that the sub-team is not necessarily a team consisting of just maintainers. Would that be accurate?

Let's discuss how this work and I can put in the necessary changes via a new PR.

### PR DESCRIPTION
**Pull request by kchristidis**
1. Replace reference to sub-teams with maintainers — addresses #2.
2. Explain reference to "RFC assignee" — closes #2.
3. Normalize references to Fabric: existing version mixes up "Hyperledger Fabric", "fabric", "Fabric"; this CR refers to the project by its full name the first time, and switches to "Fabric" for all remaining references.
4. Fix broken table of contents.
5. Remove unused or broken links.
6. Remove "help this is all too informal!" section.
7. Conform to Markdown linter directives: surround headings with blank lines, etc.
8. Enforce consistency with line breaks: existing version goes anywhere from ~80 to 100 characters, this one sticks to 80 as the upper bound with the exception lines that include links

### CODE REVIEW

**Comment by ryjones** [comment]
please squash to one commit before merge thanks

**Comment by kchristidis** [comment]
This can (and I argue _should_?) be done by the repo owner. See: https://help.github.com/en/articles/about-pull-request-merges#squash-and-merge-your-pull-request-commits

**Comment by christo4ferris** [comment]
@kchristidis why are you modifying the formatting? Having multi-line paragraphs on a single line of code makes for really awkward diff'ing/reviewing. My preference is for lines to be <80 chars.

**Comment by kchristidis** [comment]
> Having multi-line paragraphs on a single line of code makes for really awkward diff'ing/reviewing.

How? On GitHub you'll get a view similar to this no matter how wide your window is:

Screenshot 2019-10-11 at 06 38 38

On your editor you can set soft-wrap to X characters.

**Comment by kchristidis** [comment]
Preparing a patchset that address all comments and will close issue #2.

File: README.md
Comment (kchristidis) [inline README.md:210]: I removed this one as I struggle to see its usefulness. LMK if you disagree.

File: README.md
Comment (kchristidis) [inline README.md:214]: These weren't used anywhere.

File: README.md
Comment (kchristidis) [inline README.md:176]: Replaced "sub-team meeting" with "contributors meeting" (all other references to "sub-team" have been changed to "maintainers").

**Comment by kchristidis** [comment]
> please squash to one commit before merge thanks

@ryjones: Done.

**Comment by kchristidis** [comment]
> My preference is for lines to be <80 chars.

@christo4ferris: Done.

**Review by christo4ferris — APPROVED** [review]
LGTM

**Pull request merged**
